tests/workspace/__snapshots__/test_manager.ambr

```diff
@@ -0,0 +1,348 @@
+# serializer version: 1
+# name: test_if_template_is_not_rendered_during_migration[snowpark_templated_v1]
+  '''
+  defaults:
+    project_name: <! project_name | to_snowflake_identifier !>
+    stage: <! stage | to_snowflake_identifier !>
+  definition_version: '2'
+  entities:
+    hello_function:
+      artifacts:
+      - <% ctx.env.project_source %>
+      external_access_integrations: []
+      handler: functions.hello_function
+      identifier:
+        name: hello_function
+      imports: []
+      returns: string
+      secrets: {}
+      signature:
+      - arg_type: string
+        name: name
+      stage: <! stage | to_snowflake_identifier !>
+      type: function
+    hello_procedure:
+      artifacts:
+      - <% ctx.env.project_source %>
+      execute_as_caller: false
+      external_access_integrations: []
+      handler: procedures.hello_procedure
+      identifier:
+        name: hello_procedure
+      imports: []
+      returns: string
+      secrets: {}
+      signature:
+      - arg_type: string
+        name: name
+      stage: <! stage | to_snowflake_identifier !>
+      type: procedure
+    test_procedure:
+      artifacts:
+      - <% ctx.env.project_source %>
+      execute_as_caller: false
+      external_access_integrations: []
+      handler: procedures.test_procedure
+      identifier:
+        name: test_procedure
+      imports: []
+      returns: string
+      secrets: {}
+      signature: ''
+      stage: <! stage | to_snowflake_identifier !>
+      type: procedure
+  
+  '''
+# ---
+# name: test_if_template_is_not_rendered_during_migration[snowpark_templated_v1].1
+  '''
+  definition_version: "1.1"
+  snowpark:
+    project_name: "<! project_name | to_snowflake_identifier !>"
+    stage_name: "<! stage | to_snowflake_identifier !>"
+    src: <% ctx.env.project_source %>
+    functions:
+      - name: hello_function
+        handler: "functions.hello_function"
+        signature:
+          - name: "name"
+            type: "string"
+        returns: string
+    procedures:
+      - name: hello_procedure
+        handler: "procedures.hello_procedure"
+        signature:
+          - name: "name"
+            type: "string"
+        returns: string
+      - name: test_procedure
+        handler: "procedures.test_procedure"
+        signature: ""
+        returns: string
+  
+  '''
+# ---
+# name: test_if_template_is_not_rendered_during_migration[streamlit_templated_v1]
+  '''
+  definition_version: '2'
+  entities:
+    <! name | to_snowflake_identifier !>:
```

Comment:
We don't support templates as keys. This should be migrated to 
```
entity_1:
  identifier:
    name: <! name | to_snowflake_identifier !>
```

Reply:
Changed